tests/kernels/quantization/test_block_fp8.py

```diff
@@ -183,7 +184,11 @@ def test_w8a8_block_fp8_matmul(M, N, K, block_size, out_dtype, seed):
 def test_w8a8_block_fp8_fused_moe(M, N, K, E, topk, block_size, dtype, seed):
     if topk > E:
         pytest.skip(f"Skipping test; topk={topk} > E={E}")
-
+    if E == 258:
```

Comment:
From the top of the file `E = [2, 8, 16, 24]  # [128, 256]` it doesn't seem like this case is being tested?